source/_integrations/python_script.markdown

```diff
@@ -39,7 +39,7 @@ hass.bus.fire(name, {"wow": "from a Python script!"})
 ```
 
 - Start Home Assistant
-- Call service `python_script.hello_world` with parameters
+- Call service (can found at: settings-> scripts -> create new script. At the sequence area in the field action type) `python_script.hello_world` with parameters
```

Comment:
The instructions you add to the line, does not correspondent to "Call service". Could you clarify what you are trying to achieve?

Reply:
I'm trying to help new users find "services".
When you say my assumption is wrong, what would be right?
Where can I find these "services"?

Reply:
The documentation :)

Reply:
You can create a service to call it afterwards. When you have another idea write it down there.
This discussion shows that it is not clearly defined how a service should be invoked.

Reply:
Aaah! Got it I think.
So what is meant here, is that after creating you can go to the developer tools -> Services and call it manually there (for the sake of testing your new script).

Suggestion:

```suggestion
- Call your new {% my developer_call_service service="python_script.hello_world" %} service (with parameters) from the {% my developer_services %}. 
```

This will make the `python_script.hello_world` a link, directly to the service panel with the service selected. The last bit, links to the service panel itself with all the services.

Reply:
Yea, this is waht i mean.